docs/self-managed/platform-deployment/helm-kubernetes/guides/using-existing-opensearch.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
The official name is `OpenSearch`, let's use it like that.

Reply:
done